elastic/kibana
# feat(slo): improve burn rate panel

Resolves https://github.com/elastic/kibana/issues/195139

## 🌸 Summary

The history details page was using the burn rate component but only to show the error rate chart named as "burn rate", which is incorrect. I've removed the usage of that component, and only used the ErrorRateChart on that page. I was able to refactor this component slightly to remove the need to provide a `selectedTabId`.

Then, I changed the burnRatePanel component to use both the short and long window burn rates in order to handle cases like recovering/increasing burn rate.

### Testing

- Run dataforge: `node x-pack/scripts/data_forge.js --events-per-cycle 50  --lookback now-7d --dataset fake_stack --install-kibana-assets --kibana-url http://localhost:5601/kibana`
- Create a few SLOs with different objective (80% and 99.9%) for example
- Check the burn rate panel is showing the correct status based on the burn rate values returned by the API


Screenshot |
-- |
![image](https://github.com/user-attachments/assets/bb305d5a-563b-4469-b0ec-cdcfd3d4da38) |
![image](https://github.com/user-attachments/assets/e3df8a01-4130-4fa4-b55b-a839994d01df) |
![image](https://github.com/user-attachments/assets/b92c179a-0b3f-4120-b3fc-ec24436ae23b) |
![image](https://github.com/user-attachments/assets/8bb5908d-e15a-484f-9e8a-1ba63ac49d84) |
